cfme/tests/test_appliance.py

```diff
@@ -463,3 +464,44 @@ def test_appliance_log_error():
         initialEstimate: 1/2h
     """
     pass
+
+
+@pytest.mark.ignore_stream('5.9')
+def test_codename_in_log(appliance):
+    """
+    check whether logs contains a mention of appliance codename
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+    """
+    log = '/var/www/miq/vmdb/log/evm.log'
+    appliance.ssh_client.run_command('echo > {}'.format(log))
```

Comment:
I don't think this is a good way to do the test, as it erases the `evm.log` for the testrun up until this test as run. If this is the last test run on an appliance, we've just lost `evm.log` for the entire testrun.

There is a LogValidator tool for doing just what you're looking for - making sure that some content appears within the given log _only during this test method execution_. There are several uses of it in existing tests as examples.